english/33hours/wakingup.md

```diff
@@ -3,8 +3,8 @@ from the brightness. You're still in your clothes and are lying in the middle of
 cheap motel room. You have no idea where you are or how you got here.
 
 You sit up and immediately feel pain on the back of your head. Some dried blood cakes the pillow you were lying on. Your
-wallet and mobile phone are gone. In fact, your pockets are completely empty.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.